### ISSUE TITLE
Cannot find annotations/val.json

### ISSUE THREAD

**Issue by Qcmango** [fubel/synthehicle#11]
I am currently conducting an experiment on 2D vehicle detection using this dataset. I was only able to locate the annotation file for the training set (train_mono3d.coco.json). Would it be possible to share the annotation files for the test set (e.g., Town06-O-dawn) as well?

**Comment by fubel** [comment]
Hi, if I recall correctly, we did the detection experiments in yolo format, so I don't have a prepared coco-style json for the test set. You should be able to create it from the annotation files that are provided, and feel free to ask here if you need any assistance.

**Comment by Qcmango** [comment]
> Hi, if I recall correctly, we did the detection experiments in yolo format, so I don't have a prepared coco-style json for the test set. You should be able to create it from the annotation files that are provided, and feel free to ask here if you need any assistance.

Thank you so much for your prompt response! With the gt files you provided, I was able to successfully get a COCO-style JSON file. I’m happy to share that after implementing the evaluation pipeline, my reproduced results almost match the performance reported in your paper.
